### PR TITLE
[4.0] Routing not working in com_tags

### PR DESCRIPTION
**Pull request by SharkyKZ**
### Summary of Changes

Fixes routing not working in com_tags.

### Testing Instructions

Create some tags.
Go to all tags list.
Inspect tag links.

### Expected result

SEF URLs.

### Actual result

Non-SEF URLs.

### Documentation Changes Required

IDK.

### CODE REVIEW

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 0de532ca0d0f0823a071820bd01f4561f7b84e5b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27694">issues.joomla.org/tracker/joomla-cms/27694</a>.</sub>

**Comment by sakiss** [comment]
I have tested this item :white_check_mark: successfully on b05988d09f704e0b84e811516fd594f28f537fb5<br /><br />Works as described<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27694">issues.joomla.org/tracker/joomla-cms/27694</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on b05988d09f704e0b84e811516fd594f28f537fb5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27694">issues.joomla.org/tracker/joomla-cms/27694</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27694">issues.joomla.org/tracker/joomla-cms/27694</a>.</sub>

**Comment by rdeutz** [comment]
@Hackwar could you have a look here, just to be save, thanks

File: administrator/components/com_tags/tags.xml [hunk 1]
```diff
@@ -12,7 +12,6 @@
 	<namespace>Joomla\Component\Tags</namespace>
 	<files folder="site">
 		<folder>Controller</folder>
-		<filename>router.php</filename>
```
Comment (brianteeman) [inline administrator/components/com_tags/tags.xml:14]: don't you need to add
`<folder>Service</folder>`

??

Comment (SharkyKZ) [reply administrator/components/com_tags/tags.xml:14]: Added.

**Comment by richard67** [comment]
@Hackwar Could you have a look here, just to be save, thanks.

**Pull request merged**